bom/application/pom.xml

```diff
@@ -6127,6 +6128,12 @@
                 <groupId>org.eclipse.jgit</groupId>
                 <artifactId>org.eclipse.jgit</artifactId>
                 <version>${jgit.version}</version>
+	    </dependency>	
+
+            <dependency>
+                <groupId>com.graphql-java</groupId>
+                <artifactId>graphql-java</artifactId>
+                <version>${graphql-java.version}</version>
             </dependency>
```

Comment:
Why is this needed ? Will this not be pulled in from the smallrye dep ?

Reply:
This is to satisfy the enforcer plugin. Vert.x graphql extension still depends on graphql-java 19.2, so we need to get the convergence aligned. When something is pulled in by two different libraries, it's generally a good idea to manage the version manually.

Reply:
Thanks for the explanation but please keep the graphql-java version with the SmallRye GraphQL one and add a comment at the end of the line of the first one saying both should be kept in sync. Otherwise we will forget about it.

Reply:
Added a comment. I put them together inside the `<dependencyManagement>` but I don't quite like putting the `<properties>` together because it would look weird to have one different property inside ~20 properties that start with `smallrye`